### ISSUE TITLE
Unencrypted SMS not received

### ISSUE THREAD

**Issue by rstuart** [WhisperSystems/Signal-Android#7139]
I have:
- [ x] searched open and closed issues for duplicates
- [ X ] read https://github.com/WhisperSystems/Signal-Android/wiki/Submitting-useful-bug-reports

----------------------------------------

### Bug description

My wife installed Signal briefly on an S8, didn't like it, so I deregistered it. (It's still installed.) Ever since then she soarodically does not receive SMS's form the 3 family members who use Signal on their Android phones. Signal's UI definitely shows its sending an unencrypted SMS. 

### Steps to reproduce
- using hyphens as bullet points
- list the steps
- that reproduce the bug

I haven't been able to figure out what triggers it. When I get home and discover a message has not been received (the issue has long since disappeared off the debug log) and I send a test SMS ithas  always worked(!) It's worked both when using WiFi and over  3G, and it's failed in both cases too.

### Link to debug log
https://gist.github.com/anonymous/671801112c95172d0ba95a69e82b2a3f

**Comment by FeuRenard** [comment]
Duplicate of #4851?

**Comment by moxie0** [comment]
Is she using Signal as her SMS app, or a different app?

**Comment by rstuart** [comment]
She is using Google Messenger now.  Signal worked fine while it was installed.

**Comment by moxie0** [comment]
What version of Android is she running? If it's KitKat+ and Signal isn't the default SMS app, there's no way Signal could be interfering with Messenger.

**Comment by rstuart** [comment]
I don't know what version of Android it is off the top of my head, but it's a Samsung S8 so maybe N?

I don't believe it's her end that's causing at fault because AFAICT, people who don't use Signal can send SMS's to her 100% reliably.  It's only those of us who use Signal who have the occasional message go missing. It only started happening after she installed Signal for a week, the stopped using it.  The obvious thought is Signal isn't sending the unsecured message via SMS - but if so, the UI on my phone shows no sign of it.

I'm provided my debug log which captures what happened when I sent a message that wasn't received.

**Comment by moxie0** [comment]
@rstuart From that log, Signal did send an SMS:

`````
10-30 18:03:47.630  8708  8789 W SmsSentJob: Got SMS callback: org.thoughtcrime.securesms.SendReceiveService.SENT_SMS_ACTION , -1
`````

Signal doesn't really have any control over SMS delivery, we just hand it to the OS and process whatever callbacks it gives us. If it shows a single tick or delivery receipt, that means the OS says it has been sent/delivered via SMS.

**Comment by jeremymasters** [comment]
@moxie0 https://www.sammobile.com/2017/10/06/galaxy-s8-missing-texts-issue-irks-users/

**Comment by rstuart** [comment]
New data arrived last night: wife has now reported an SMS sent the conventional way was lost.  So the major reason I thought it was Signal (ie, only SMS's sent by Signal were being lost) has gone.   I'm closing the bug.  Sorry for the noise.

**Comment by PrzemyslawDuda** [comment]
Hi, I will throw some salt into the open wound...
I am having similar issue on Xiaomi MI5 MIUI 9.1.1.0. I've flashed it today via my own hands with a official ROM from the Xiaomi. I've set permissions so only signal is able to touch SMS. My observation is that normal chat-related SMSes arrivies smoothly (but while my phone is idle, signal does not notify me when something arrives, which makes this app on this particular software useless :( ) but funny thing is that when I'm sending some "passwords" like (and unencrypted) SMSes, they disapper. 

It sounds odd, isn't it?
